src/Raven.Server/Documents/PeriodicBackup/Restore/AbstractRestoreBackupTask.cs

```diff
@@ -63,6 +63,7 @@ public abstract class AbstractRestoreBackupTask : IDisposable
             RestoreSource = restoreSource;
             FilesToRestore = filesToRestore;
             OperationCancelToken = operationCancelToken;
+            ServerStore.ContextPool.AllocateOperationContext(out Context);
```

Comment:
this returns disposable that you need to dispose instead of context

Reply:
right, thanks
fixed